packages/admin-next/dashboard/src/routes/inventory/inventory-create/components/create-inventory-item-form/schema.ts

```diff
@@ -0,0 +1,23 @@
+import { z } from "zod"
+import { optionalInt } from "../../../../../lib/validation"
+
+export const CreateInventoryItemSchema = z.object({
```

Comment:
question: Isn't this a duplicate of this [schema](https://github.com/medusajs/medusa/blob/e1faaff6eaeadc3a16d85292fcf69060c76bad86/packages/admin-next/dashboard/src/routes/inventory/inventory-create/components/inventory-create-form/schema.ts)? Or the other way around.

Reply:
It seems like the files got copied over, as I renamed them in this PR and changed them in another. Deleted the duplicate files now 👍